app/src/main/java/org/mozilla/fenix/browser/BrowserFragment.kt

```diff
@@ -630,11 +630,15 @@ class BrowserFragment : Fragment(), BackHandler {
         sessionManagerObserver?.let {
             requireComponents.core.sessionManager.unregister(it)
         }
+        pendingIntent?.let {
```

Comment:
Let's use a better name for the pendingIntent. It's important to clarify to other developers that we're not using this for a typical purpose, like popping a notification. This is a workaround to avoid having two activities attempting to access the same session concurrently and crashing GeckoView.